tests/pytests/standard_suite_ref_local.py

```diff
@@ -0,0 +1,300 @@
+import numpy as np
+from qcengine.programs.tests.standard_suite_ref import answer_hash, _std_suite, _std_generics
+
+
+_std_suite_psi4_extension = [
```

Comment:
Please include a comment explaining how to add items to this file and why.

Reply:
It's key/value pairs to extend data of dict keys in the same format as the qcng file at the top of this file, https://github.com/MolSSI/QCEngine/blob/master/qcengine/programs/tests/standard_suite_ref.py#L447. I'm not sure that extra comments help more than the unavoidable reading/searching.

I agree stdsuite as a whole could use some procedural documentation (what repos to open and edit, what options to have engaged, what sub-checks to disengage at first, what tests to set up for a mtd) that I'm rather at a loss to know where to start. This part, though, is navigable by search-and-extend.

Reply:
You're the only developer who knows how the standard suite machinery works. Can you at least document somewhere _when_ to add an entry to this file?

Reply:
Yes, I can add the below in a future PR.

```
# in-repo extensions for _std_suite above
# * ideally empty. PR to QCEngine ASAP and empty this after QCEngine release.
_std_suite_psi4_extension = [
```